stacker_blueprints/generic.py

```diff
@@ -0,0 +1,92 @@
+from troposphere import (
+    Ref, Output
+)
+
+from stacker.blueprints.base import Blueprint
+from stacker.util import load_object_from_string
+
+
+class GenericResourceCreator(Blueprint):
+    """ Generic Blueprint for creating a resource """
+    def add_cfn_description(self):
+        """ Boilerplate for CFN Template """
+        template = self.template
+        template.add_version('2010-09-09')
+        template.add_description('Generic Resource Creator - 1.0.0')
+
+    """
+
+    *** NOTE ***    Template Version Reminder
+
+    Make Sure you bump up the template version number above if submitting
+    updates to the repo. This is the only way we can tell which version of
+    a template is in place on a running resouce.
+
+    """
+
+    VARIABLES = {
+        'Class':
+            {'type': str,
+             'description': 'The troposphere class to create, '
+                            'e.g.: ec2.Volume'},
+        'Output':
+            {'type': str,
+             'description': 'The output field that should be created, '
+                            'e.g.: VolumeId'},
+        'Properties':
+            {'type': dict,
+             'description': 'The list of properties to use for the '
+                            'Troposphere class'},
+    }
+
+    """
```

Comment:
Can you just combine this with the docstring for the blueprint itself?  That way if we (hopefully someday) enable sphinx API documentation this will be auto-formatted into it.